### ISSUE TITLE
A question about correspendence Loss

### ISSUE THREAD

**Issue by sugerkiller** [Jumpat/SegAnyGAussians#10]
Thank you for open-sourcing your codes，When I was reading the code I found a problem，The code location is in "train_contrastive_feature.py"
```jsx
        norm_rendered_feature = torch.nn.functional.normalize(torch.nn.functional.interpolate(rendered_features.unsqueeze(0), (H,W), mode = 'bilinear').squeeze(), dim=0, p=2)
        correspondence = torch.relu(torch.einsum('CHW,CJK->HWJK', norm_rendered_feature, norm_rendered_feature))
```
Two identical "norm renderfeature" matrices are multiplied to get the result called "correspondence". I don't understand this step. Can you help me explain it?

**Comment by Jumpat** [comment]
Hello, this is the calculation of the feature correspondence: N*C, M*C -> N*M, which indicates the feature similarities between each pair of features.
